kubernetes/parser.py

```diff
@@ -0,0 +1,50 @@
+from influx_line_protocol import Metric, MetricCollection
+
+
+class Parser(object):
+
+    def parse_stats(self, stats):
+        collections = MetricCollection()
+
+        for pod in stats["pods"]:
+            m = Metric("kubernetes_pod")
+            m.add_tag("pod", pod["podRef"]["name"])
+            m.add_value("cpu_usage", pod["cpu"]["usageNanoCores"] / 1000000000)
```

Comment:
`1000000000` moving it to the constant can help a lot with readability, right now it's like a magic number here